src/synced_context.cpp

```diff
@@ -341,6 +341,7 @@ config synced_context::ask_server_choice(const server_choice& sch)
 
 			//here we can get into the situation that the decision has already been made but not received yet.
 			synced_context::pull_remote_user_input();
+			//FIXME: we shoudl call play_controller::play_silce or the application will freeze while waiting for a remote choice.
```

Comment:
I think it is, but its not caused by this pr so it should be fixed unrelated to this or.